docs/autogenerate/dummy.nix

```diff
@@ -0,0 +1,19 @@
+builtins.mapAttrs (k: _v:
+  let
+    url = "https://github.com/NixOS/nixpkgs/archive/2255f292063ccbe184ff8f9b35ce475c04d5ae69.tar.gz";
+    pkgs = import (builtins.fetchTarball url) { system = k; };
+  in
+  pkgs.recurseIntoAttrs {
```

Comment:
AFAIK this is redundant.

Reply:
The attrs look like x86_64-darwin.hello, so we do have 1 level of nesting.